services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -149,58 +134,19 @@ async def create_computation(
 
         if job.start_pipeline:
             if not computational_dag.nodes():
+                # 2 options here: either we have cycles in the graph or it's really done
+                list_of_cycles = find_computational_node_cycles(complete_dag)
+                if list_of_cycles:
+                    raise HTTPException(
+                        status_code=status.HTTP_403_FORBIDDEN,
+                        detail=f"Project {job.project_id} contains cycles with computational services which are currently not supported! Please remove them.",
```

Comment:
yes. was thinking about it but it would need some more changes in the frontend and the small ones in the webserver. This is beyond the scope of this PR. I created a separate issue for this: #2351

Reply:
MINOR: can perhaps also add these errors in the ``@router.post`` decorator so it appears in the directorv2 API specs ie.

E.g.
```python
@router.post(
   # ...
    responses={
        status. HTTP_403_FORBIDDEN: {"description": "Unsupported CS"},
    },)
   #...
```

Reply:
no you get a 403 for different reasons.
I think the best way is to return where the issues are in the TaskOut structure. there might also be other issues.